pkg/controller/clusteringress/resources/route_test.go

```diff
@@ -24,15 +129,10 @@ func TestMakeRoute(t *testing.T) {
 		Status: networkingv1alpha1.IngressStatus{
 			LoadBalancer: &networkingv1alpha1.LoadBalancerStatus{
 				Ingress: []networkingv1alpha1.LoadBalancerIngressStatus{{
-					DomainInternal: "istio-ingressgateway.istio-system.svc.cluster.local",
+					DomainInternal: domainInterval,
```

Comment:
nit: this variable should be `domainInternal` and not `domainInterval` as it's the internal domain, not a time interval.

Reply:
Changed and updated